### PR TITLE
User can not be deleted in LDAP authentication mode.

### PR DESCRIPTION
**Pull request by ywk253100**

### CODE REVIEW

File: src/ui/api/user.go [hunk 1]
```diff
@@ -241,6 +241,12 @@ func (ua *UserAPI) Delete() {
 		return
 	}
 
+	// TODO read from conifg
+	authMode := os.Getenv("AUTH_MODE")
+	if authMode == "ldap_auth" {
+		ua.CustomAbort(http.StatusMethodNotAllowed, "user can not be deleted in LDAP authentication mode")
```
Comment (reasonerjt) [inline src/ui/api/user.go]: Let's just use 403 here, 405 should container a "Allow" header in the response.

**Comment by wknet123** [comment]
Part of UI has been fixed with #974

**Comment by reasonerjt** [comment]
@ywk253100, sorry, I see you removed that chunk?

**Comment by ywk253100** [comment]
@reasonerjt Updated.

**Pull request merged**